pkgs/development/libraries/libnabo/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchgit, cmake, eigen, boost}:
+
+stdenv.mkDerivation rec {
+  version = "1.0.6";
+  name = "libnabo-${version}";
+
+  src = fetchgit {
+    url = "git://github.com/ethz-asl/libnabo.git";
+    rev = "2df86e0ecdd712e9d0fdd9f268610156263a88f1";
+    sha256 = "14w2jz9fvc45lc8i23y8a7lqawq132swl7lha5dhlwrlfj0cg82d";
+  };
+
+  buildInputs = [cmake eigen boost];
+
+  enableParallelBuilding = true;
+
+  inherit eigen;
```

Comment:
I copied this from another package because cmake couldn't find eigen. I'll try without the `inherit`.